modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -10,35 +10,99 @@ You can share subnets within a configured VPC with another AWS user account if t
 
 .Procedure
 
-. From the AWS account that centrally manages your VPC, create or modify a VPC to your specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. This AWS account will be the *VPC-owning AWS account*. 
-. In the link:https://us-east-1.console.aws.amazon.com/iamv2/[Identity and Access Management (IAM) section of the AWS console], create a custom trust policy role for the shared VPC permissions. This role needs to have the following permissions:
-  * A trust policy to assume roles:
+. From the AWS account that centrally manages your VPC, create or modify a VPC to your specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. This AWS account will be the *VPC-owning AWS account*.
++
+. Create a custom policy file to allow for necessary Shared VPC permissions that uses the name `SharedVPCPolicy`:
 +
 [source,terminal]
 ----
+$ cat <<EOF > /tmp/shared-vpc-policy.json
 {
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-	  "Sid": "Statement1",
-	  "Effect": "Allow",
-	  "Principal": {
-	  	"AWS": "arn:aws:iam::<Account-ID>:root"
-	  }, <1>
-	  "Action": "sts:AssumeRole"
-	}
-  ]
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "route53:ChangeResourceRecordSets",
+                "route53:ListHostedZones",
+                "route53:ListHostedZonesByName",
+                "route53:ListResourceRecordSets",
+                "route53:ChangeTagsForResource",
+                "route53:GetAccountLimit",
+                "route53:GetChange",
+                "route53:GetHostedZone",
+                "route53:ListTagsForResource",
+                "route53:UpdateHostedZoneComment",
+                "tag:GetResources",
+                "tag:UntagResources"
+            ],
+            "Resource": "*"
```

Comment:
@oriAdler should we restrict the resource in this permission?

Reply:
Not sure, if the user decides to use the same role with this policy for multiple clusters, in other words for multiple hosted zones, we can't restrict it to a single hosted zone resource.